docs/terms-of-service.rst

```diff
@@ -15,124 +15,151 @@ Last changed: |date|
 Signing up
 ----------
 
-#. In order to use the UH-IaaS infrastructure service, you must sign
-   up with the Dataporten authentication and authorization service.
+.. _Dataporten: https://www.uninett.no/en/
 
-#. You must have a valid user account at any educational institution
-   in Norway that has signed up for using UH-IaaS.
+* In order to use the UH-IaaS infrastructure service, you must sign
+  up with the Dataporten_ authentication and authorization service.
 
```

Comment:
Har et problem med denne endringen. Det at institusjonen har koblet seg på dataporten er ikke noe brukeren trenger å forholde seg til. Det må fikses før vi lar dem ta i bruk tjenesten (typisk når de signerer databehandleravtale).

Reply:
Endre Dataporten til Feide? For brukeren må logge seg inn via feide for autentisering

Reply:
Det har ikke noe med det gå gjøre. Dette er jo et nytt punkt såvidt jeg kunne se?

Prøver å forstå hvorfor vi skriver noe om Dataporten i ToS. Vi kan jo skrive at bruker må godta at Dataporten deler epost og navn med oss, men det står vel under privacy alt?